File: 1 Klasa/KartyPracy/Kartapracy 2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace KartaPracy2
{
    internal class cs
    {
        static void Main(string[] args)
        {
            {//zad 1
                int a;
                a = int.Parse(Console.ReadLine());
                if (a % 3 == 0)
                {
                    Console.WriteLine("Tak");
                }
                else
                {
                    Console.WriteLine("Nie");
                }
                //zad 2
                a = int.Parse(Console.ReadLine());
                if (a >= 100 && a < 1000 && a % 17 == 0)
                {
                    Console.WriteLine("Tak");
                }
                else
                {
                    Console.WriteLine("Nie");
                }
                //zad 3
                a = int.Parse(Console.ReadLine());
                if (a >= 18)
                {
                    Console.WriteLine("Tak");
                }
                else
                {
                    Console.WriteLine("Nie");
                }
                //zad 4
                const int waga = 20;
                a = int.Parse(Console.ReadLine());
                if (a >= waga)
                {
                    Console.WriteLine("Nie");
                }
                else
                {
                    Console.WriteLine("Tak");
                }
                //zad 5
                a = int.Parse(Console.ReadLine());
                int b = int.Parse(Console.ReadLine());
                int c = int.Parse(Console.ReadLine());
                if (a < c && c < b || a > c && c > b)
                {
                    Console.WriteLine("Tak");
                }
                else
                {
                    Console.WriteLine("Nie");
                }

                //zad 6
                a = int.Parse(Console.ReadLine());
                int p = int.Parse(Console.ReadLine());
                if ((Math.Pow(a, p) - a) % p == 0) //=if (a**p - a) % p == 0:
                {
                    Console.WriteLine("Tak");
                }
                else
                {
                    Console.WriteLine("Nie");
                }
                //zad 7
                p = int.Parse(Console.ReadLine());
                int k = int.Parse(Console.ReadLine());
                int s = int.Parse(Console.ReadLine());
                if (s * 3 >= k - p) //=if (k-p) <= 3*s:
                {
                    Console.WriteLine("Tak");
                }
                else
                {
                    Console.WriteLine("Nie");
                }
                //&&=and
                //||=or

                Console.ReadKey();
            }
        }

    }

}   

File: 1 klasa/KartyPracy/Kartapracy2.py
#zad 1
a = int(input())
if a % 3 == 0:
  print("tak")
else:
  print('nie')
#zad 2
a = int(input())
if a % 17 == 0 and a >= 100 and a < 1000:
  print('tak')
else:
  print('nie')
#zad 3
wiek = int(input())
if wiek >= 18:
  print('tak')
else:
  print('nie')
#zad 4
limit = 20
waga = int(input())
if waga <= limit:
  print('tak')
else:
  print('nie')
#zad 5
a, b, c=map(int, input().split())
a = int(input())
b = int(input())
c = int(input())
if a < c < b or a > c > b:
  print("TAK")
else:
  print("NIE")
#zad 6
a = int(input("Podaj dowolna liczbe: "))
p = int(input("Podaj liczbe pierwsza: "))
if (a**p - a) % p == 0:
  print("tak")
else:
  print("nie")
#zad 7
p, k, s=map(int, input().split())
if (k-p) <= 3*s:
  print('tak')
else:
  print('nie')